olive/cli/quantize.py

```diff
@@ -157,16 +254,10 @@ def run(self):
 }
 
 ALGORITHMS = {
-    "awq": (["awq"], "(HfModel) int4 WOQ with AWQ."),
-    "gptq": (["gptq"], "(HfModel) int4 WOQ with GPTQ."),
-    "quarot": (["quarot"], "(HfModel) QuaRot/Hadamard rotation + AWQ/RTN quantization."),
-    "bnb4": (["bnb4"], "(OnnxModel) nf4 WOQ using bitsandbytes."),
-    "int4_rtn": (["matmul4", "mnb_to_qdq"], "(OnnxModel) int4 WOQ with RTN using onnxruntime. DQ->MatMul is used."),
-    "int4_rtn_mnb": (
-        ["matmul4"],
-        "(OnnxModel) int4 WOQ with RTN using onnxruntime. MatMulNBits contrib is used instead of DQ->MatMul.",
-    ),
-    "nvmo": (["nvmo"], "(OnnxModel) int4 WOQ with RTN using Nvidia-ModelOpt. DQ->MatMul is used."),
-    "onnx_dynamic": (["onnx_dynamic"], "(OnnxModel) Dynamic quantization using onnxruntime."),
-    "inc_dynamic": (["inc_dynamic"], "(OnnxModel) Dynamic quantization using neural-compressor."),
+    "awq": (["awq", "inc_static", "inc_dynamic"], "(HfModel, OnnxModel) WOQ with AWQ."),
```

Comment:
can you test the command manually on an actual model to make sure the cli is not broken? the ut mocks the run so it will pass. 

the first element is used to keep the pass_flow for each algorithm. you added multiple passes here which cannot be run on pytorch or chained together.

Reply:
oh, just saw the new line 159. this first part in the tuple is not needed anymore then.